### ISSUE TITLE
segmentation get worst after training from default cellpose model

### ISSUE THREAD

**Issue by FrenchSauce** [MouseLand/cellpose#349]
Hello,

I have tried to train a cellpose model either from scratch or using a pretrained model from cellpose (the nuclei model) but no matter what I try, the result is terrible, here is an overview: 

![image](https://user-images.githubusercontent.com/88531215/137917733-6cc67515-29f8-444c-9f8d-315010ccf000.png)

I can understand why the segmentation is worst after training from scratch, but how could it get worst after training on top of the default nuclei model (which performs quite ok)? Am I missing something? I have used 16bits masks and 16bits images, could it be the issue?

I have used the following parameters for the training:

model_to_load = 'nuclei'
number_of_epochs = 500
initial_learning_rate = 0.0002
batch_size = 8
diameter = 30 (also tried to set 0)

and the following code to train the model:
`python -m cellpose --train --dir "$train_folder" --pretrained_model $model_to_load --chan $channel --n_epochs $number_of_epochs --learning_rate $initial_learning_rate --batch_size $batch_size --use_gpu`

Thanks!

**Comment by carsen-stringer** [comment]
there was a bug in training for single channel images like yours, I'd recommend trying the latest version of the code if you haven't already (`pip install git+https://github.com/mouseland/cellpose.git`)

I'm going to close this issue for now because I'm hoping that the bug is the problem, if not let us know

In general I've also found that retraining from scratch works better than training from the default model

**Comment by wheelern** [comment]
I'm running into this same problem (segmentation performing much worse after training on single-channel images). In fact, my trained model finds 0 masks, even when LC4 routinely finds ~80% of the objects. This suggests (to me) that I'm doing something incorrectly or a bug remains. I've followed the guidelines on the human-in-the-loop training, and everything proceeds without error...

Happy to share my images and the model itself if that's helpful. FWIW, I'm training on unlabeled _C. elegans_.

**Comment by Nelson-Gon** [comment]
Looking at this, is retraining from scratch better almost always or is this specific to the nuclei pretrained model? Thanks
>In general I've also found that retraining from scratch works better than training from the default model

**Comment by MartinKahn** [comment]
Hi, thanks for making cellpose!  I seem to have the same problem of training doing worse than the basic cyto 2 model. In fact after a while (went up to 10 rounds) it doesnt detect any cells any more in some images. I'm doing this with the current branch installed via git+pip https... . I tried it in the GUI and in the notebook with various parameters. 

Im running just a single channel without DAPI. Is this the problem? Happy to share some example tiffs and hand-segmented segments.

**Comment by Myrkgod** [comment]
Was this issue ever figured out? I am currently using an up-to-date program and am also seeing similar bad performance. https://forum.image.sc/t/high-loss-when-training-cellpose-models/79937?u=myrk
